airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -737,7 +737,7 @@ def run_query(self,
         :type labels: dict
         :param schema_update_options: Allows the schema of the destination
             table to be updated as a side effect of the query job.
-        :type schema_update_options: tuple
+        :type schema_update_options: list
```

Comment:
```suggestion
        :type schema_update_options: list or tuple
```
it seems to me that two types are supported here.

Reply:
In fact this could be set, list or tuple. Main problem is that there is a type validation preformed in `run_query` method.